bundle/edu.gemini.spModel.io/src/main/scala/edu/gemini/spModel/io/PioSyntax.scala

```diff
@@ -21,11 +24,13 @@ object PioSyntax {
 
     def findContainers(spc: SPComponentType): List[Container] =
       allContainers.filter(_.componentType.contains(spc))
-
   }
 
   implicit class ContainerOps(c: Container) {
 
+    def nodeKey: Option[SPNodeKey] =
+      Try(new SPNodeKey(c.getKey)).toOption
```

Comment:
Just curious why we're using `Try` here, instead of:

`\/.fromTryCatchNonFatal[SPNodeKey](c.getKey)).toOption`
?